src/ui/suspense/frameCache.ts

```diff
@@ -60,3 +82,108 @@ function createPauseFrames(pauseId: PauseId, frames: Frame[], sourcesState: Sour
     sourcesState.sourceDetails.entities
   )!;
 }
+
+export interface FormattedPointStackFrame {
+  url?: string;
+  source: Source;
+  functionLocation: Location;
+  executionLocation: Location;
+  point: PointDescription;
+  functionDetails?: FormattedEventListener;
+}
+
+export interface FormattedPointStack {
+  point: PointDescription;
+  resultPoint: PointDescription;
+  frame?: FormattedPointStackFrame;
+  allFrames: FormattedPointStackFrame[];
+  filteredFrames: FormattedPointStackFrame[];
+  functionName?: string;
+}
+
+export async function formatPointStackFrame(
```

Comment:
Same question about this function and only being used in one place. Can we either inline it into the cache or at least not export it?